### DISCUSSION TITLE
USDT vanished during wormhole bridge from SOL to ETH

### DISCUSSION THREAD

**Discussion by Sandybeach25** [wormhole-foundation/wormhole#450] [Problems / bugs] [answered]
Dear Solana DEV(s), please help me to recover my vanished USDT!
What happened is this:
I bridged USDT fromm Solana to Ethereum with Solana’s Wormhole bridge.
I used the Phantom wallet at the SOL side, and my Metamask at the ETH side.
I use metamask to connect to my hardware wallet.
During the bridge I was uncertain about one button at the bottom that said “close your Ethereum wallet”. I decided to not close that, and click on REDEEM.
Al looked well, and I signed and paid for the transaction in Metamask.

In the Etherscan it looks all good. Here:
https://etherscan.io/tx/0x871d0c55abb37290bae1e964571bb8908895d68132f2eb675fda29c866367922

But in the Solana explorer:
https://explorer.solana.com/tx/33L3CcKnAZshPotD1A72L7x1Tcsfg85HUQisuDSiGv6KWwuzdFjecc1tkv8h4HJyHP9zAtnEQ7UKnpuqqxtqbFcm
I noticed there is next to my Solana address something highlighted in purple/pink.

The problem now is that my USDT is gone from my Solana address AND no USDT has arrived in my Ethereum address (zero!)
If it is so that I should add something like a wrapped USDT wormhole-address in Metamask I urge you to inform me how to do that. Leo, one of your dev’s who tried to help me in Github told me that he also doesn’t know how to add such a address in Metamask.

Leo, told me that he thought the bridge went well, but that probably there is no liquidity so I can have no use of my USDT in the Ethereum chain, and he advised me to bridge back the USDT to Solana chain.
BUT!!!! 
That is impossible because there is zero USDT visable in my Metamask eth-address!

Please help me to find back my USDT that now seems to be gone up in smoke between SOL-chain and ET-chain during the bridge.

I tried to REPAIR (because it is ann option in wormhole), but that delivered this screen:
![image](https://user-images.githubusercontent.com/90960025/133896060-29db39b3-819a-4034-80f4-be005e60313d.png)

Please let me know what to do!
I will thank you until the end of times!
Marco

PS:
I just discovered that when looking at my eth-address there is listed that i have the amount of USDT that i bridged, in an asset that is called WORMHOLE??????!!!!!!!!
https://ethplorer.io/address/0x4196694b8617186d09fb14115299d10eb206d722

**Comment by quentinadam** [comment, marked as answer]
Hello @Sandybeach25

Disclaimer: I am not a Wormhole developper but a regular user.
Shortly put, the Wormhole bridge cannot be used to bridge USDT from Solana to Ethereum. You will end up, on the Ethereum blockchain, with an asset that is not USDT, but a Wormhole wrapped version of it. As a matter of fact you did receive 922 of these Wormhole wrapped USDT tokens on Ethereum (token address https://etherscan.io/token/0x1cdd2eab61112697626f7b4bb0e23da4febf7b7c).

What I suggest you to do (which I believe you have now done), is to use the Wormhole bridge in the other direction to convert back the wormhole wrapped USDT tokens from Ethereum to USDT on Solana.

Once you have the USDT on Solana, the best way to send them to the Ethereum blockchain is to open an account on FTX.com and to bridge them through FTX.

**Reply by leoluk** [reply]
This is correct. We also added a warning when attempting to transfer USDT.
